locations/spiders/costco_global.py

```diff
@@ -7,11 +7,12 @@
 from locations.categories import Categories, Extras, apply_category, apply_yes_no
 from locations.dict_parser import DictParser
 from locations.pipelines.address_clean_up import merge_address_lines
+from locations.spiders.costco_ca_gb_us import COSTCO_SHARED_ATTRIBUTES
 
 
 class CostcoGlobalSpider(Spider):
     name = "costco_global"
-    item_attributes = {"brand_wikidata": "Q715583"}
+    item_attributes = COSTCO_SHARED_ATTRIBUTES
```

Comment:
This breaks brand in places like JP.

Reply:
Hmmm, `costco_global` will need some changes to handle native brand/name fields in JP, ZH and TW. I can submit a PR